src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -151,7 +151,7 @@ class BinaryCompactObject : public DomainCreator<3> {
   using Affine = CoordinateMaps::Affine;
   using Affine3D = CoordinateMaps::ProductOf3Maps<Affine, Affine, Affine>;
   using Identity2D = CoordinateMaps::Identity<2>;
-  using Translation = CoordinateMaps::ProductOf2Maps<Affine, Identity2D>;
+  using Translation = CoordinateMaps::ProductOf3Maps<Affine, Affine, Affine>;
```

Comment:
For backwards compatibility with existing data files you may only _add_ to the `maps_list` below, never remove or change any maps. So, define a new translation map alias here and add the new maps to the list below.

Reply:
I'm using `Affine3D` instead of changing `Translation` now. I still have to fix some domain failed tests, but `SolveXcts` is currently working.